docs/modules/extensions/pages/block-example.adoc

```diff
@@ -0,0 +1,109 @@
+= Block extension example
```

Comment:
I'm thinking about this.... so far I like my example.  Possibly it can be made simpler, I think removing the 'role' attribute is unnecessary.

Reply:
The code now could not be simpler IMO :-).  Even if you don't understand why css classes might be interesting, you can see in the output that the blocks turn into "openblock". Since all the examples are html I think that's sufficient. WDYT?

Reply:
Sure, looks fine.

Reply:
I also think that the opening example should be the shout block. It is the simplest functional example I could come up with and I think it gives the user a quick win. I'm fine with advancing on to the nestable block example right after that. You could say something like "with that under your belt, let's look at a slightly more complex example". This is a very typical progression. But we can't just throw the reader into the deep end right out of the gate.

Reply:
Compared with the shout block, the nestable block is fewer lines of code, simpler, and useful. How is this the deep end?  And how mixed are your metaphors :-)? However, this isn't worth arguing about.